CS483/CS483/Kartaclysm/Components/ComponentKartController.cpp

```diff
@@ -633,6 +634,23 @@ namespace Kartaclysm
 			m_pGameObject->GetTransform().SetRotation(glm::quat(glm::vec3(m_fOffroadRumble, m_fDirection + GetRotationMod(), 0.0f)));
 		}
 
+		std::vector<HeatStroke::GameObject*> pModels = m_pGameObject->GetChildren();
+		for (int i = 0; i < pModels.size(); i++)
+		{
+			Component* pRenderable = pModels[i]->GetComponent("GOC_Renderable");
```

Comment:
To my knowledge, the only renderables we should have are the Kart and Driver gameobjects, which the ComponentRacer contains a pointer to each. Rather than doing a component (string) search on each child, you *should* be able to access everything through just the ComponentRacer